hooks/use-hypercert-exchange-client.tsx

```diff
@@ -0,0 +1,88 @@
+"use client";
+import { useEffect, useState } from "react";
+
+import { getEthersProvider, getEthersSigner } from "@/lib/ethers";
+import { HypercertExchangeClient } from "@hypercerts-org/marketplace-sdk";
+import { useQuery } from "@tanstack/react-query";
+import { useAccount, useConfig, useWalletClient } from "wagmi";
+import { useEthersProvider } from "./use-ethers-provider";
+import { useEthersSigner } from "./use-ethers-signer";
+
+const isSupportedChain = (chainId: number) => {
+	const supportedChainIds = [10, 42220, 11155111]; // Replace with actual chain IDs
```

Comment:
should be able to import the chains from "viem/chains" and then reference the id. ex sepolia.id

Reply:
whole file isn't used so rip